clean.go

```diff
@@ -187,6 +187,10 @@ var preservedDirectories = []string{
 	// gometalinter vendors its own linters and relies on this directory's
 	// existence. See issue #7.
 	filepath.Join("github.com", "alecthomas", "gometalinter", "_linters"),
+
+	// sqlboiler requires template files to exist at runtime
```

Comment:
nit: Please end the comment with a period (`... to exist at runtime.`), and please add a reference to this PR (`See pull request #25.`).